rbac/common/role/relationship_admin.py

```diff
@@ -56,3 +56,13 @@ def related_type(self):
     def relationship_type(self):
         """The related type from AddressSpace implemented by this class"""
         return addresser.RelationshipType.ADMIN
+
+    @property
+    def _state_object_name(self):
+        """Task relationship state object name ends with Relationship (TaskRelationship)"""
+        return self._name_camel + "Relationship"
+
+    @property
+    def _state_container_list_name(self):
```

Comment:
Why do you have the same function returning a string "relationship" in two different classes?  Here